Model/Smartcalcs.php

```diff
@@ -179,28 +171,29 @@ public function getTaxForOrder(
         $order = $this->_getOrder($quote, $quoteTaxDetails, $address);
 
         if ($this->_orderChanged($order)) {
-            $client = $this->clientFactory->create();
+            $client = new LaminasClient(null, ['timeout' => 30]);
             $client->setUri($this->taxjarConfig->getApiUrl() . '/magento/taxes');
-            $client->setConfig([
+            $client->setOptions([
                 'useragent' => $this->tjHelper->getUserAgent(),
                 'referer' => $this->tjHelper->getStoreUrl()
             ]);
             $client->setHeaders([
                 'Authorization' => "Bearer $apiKey",
                 'x-api-version' => TaxjarConfig::TAXJAR_X_API_VERSION
             ]);
-            $client->setRawData(json_encode($order), 'application/json');
+            $client->setRawBody(json_encode($order));
+            $client->setEncType('application/json');
 
             $this->logger->log('Calculating sales tax: ' . json_encode($order), 'post');
 
             $this->_setSessionData('order', json_encode($order, JSON_NUMERIC_CHECK | JSON_PRESERVE_ZERO_FRACTION));
 
             try {
-                $response = $client->request('POST');
+                $response = $client->setMethod('POST')->send();
                 $this->response = $response;
                 $this->_setSessionData('response', $response);
 
-                if (200 == $response->getStatus()) {
+                if (200 == $response->getStatusCode()) {
```

Comment:
To keep referencing `\Laminas\Http\Response::STATUS_CODE_200` rather than `200`.